kivy/uix/behaviors/resize.py

```diff
@@ -0,0 +1,383 @@
+        if self.hovering:
+            if self.cursor:
+                self.cursor.on_mouse_move(pos)
+            if not self.resizing:
+                if not self.collide_point(pos[0], pos[1]):
+                    self.hovering = False
+                    self.on_leave()
+                    self.on_leave_resizable()
+                else:
+                    chkr = self.check_resizable_side(pos)
+                    if chkr != self.hovering_resizable:
+                        self.hovering_resizable = chkr
+                        if chkr:
+                            self.on_enter_resizable()
+                            self.cursor.hidden = False
+                        else:
+                            self.on_leave_resizable()
+                            self.cursor.hidden = True
+        else:
+            if self.collide_point(pos[0], pos[1]):
+                self.hovering = True
+                if not self.resizing:
+                    self.check_resizable_side(pos)
+                self.on_enter()
+
+    def check_resizable_side(self, mpos):
+        if self.resizable_left:
+            self.resizing_left = False
+            if mpos[0] > self.pos[0]:
+                if mpos[0] < self.pos[0] + self.rborder:
+                    self.resizing_left = True
+        if self.resizable_right and not self.resizing_left:
+            self.resizing_right = False
+            if mpos[0] < self.pos[0] + self.width:
+                if mpos[0] > self.pos[0] + self.width - self.rborder:
+                    self.resizing_right = True
+        if self.resizable_down:
+            self.resizing_down = False
+            if mpos[1] > self.pos[1]:
+                if mpos[1] < self.pos[1] + self.rborder:
+                    self.resizing_down = True
+        if self.resizable_up and not self.resizing_down:
+            self.resizing_up = False
+            if mpos[1] < self.pos[1] + self.height:
+                if mpos[1] > self.pos[1] + self.height - self.rborder:
+                    self.resizing_up = True
+        if self.cursor:
+            self.cursor.change_side(self.resizing_left, self.resizing_right, self.resizing_up, self.resizing_down)
+        if self.resizing_left or self.resizing_right or self.resizing_up or self.resizing_down:
+            return True
+        else:
+            return False
+
+    def on_touch_down(self, touch):
+        if self.hovering:
+            if any([self.resizing_right, self.resizing_left, self.resizing_down, self.resizing_up]):
+                self.oldpos = list(self.pos)
+                self.oldsize = list(self.size)
+                self.resizing = True
+                Window.show_cursor = False
+                return True
+            else:
+                return super(ResizableBehavior, self).on_touch_down(touch)
+        else:
+            return super(ResizableBehavior, self).on_touch_down(touch)
+
+    def on_touch_move(self, touch):
+        if self.resizing:
```

Comment:
def on_touch_move(self, touch):
        if not self.resizing:
            return super(...)
        <the rest of code that would be in "if self.resizing">